src/main/resources/config.properties

```diff
@@ -21,4 +21,5 @@ rabbitmq_routingKey=x
 mail_bcc=x
 
 siteName=India Biodiversity Portal
-serverUrl=http://indiabiodiversity.org
\ No newline at end of file
+serverUrl=http://127.0.0.1:8081/
```

Comment:
change serverUrl to http://indiabiodiversity.org

Reply:
Corrected.